Apps/WebClient/src/ClientApp/src/models/laboratoryOrderTimelineEntry.ts

```diff
@@ -69,7 +69,7 @@ export default class LaboratoryOrderTimelineEntry extends TimelineEntry {
         this.sortResults();
 
         this.downloadLabel = "Incomplete";
-        if (this.tests.every((test) => test.status === statusFinal)) {
+        if (this.tests.every((test) => test.status === statusComplete)) {
```

Comment:
This probably won't build - the variable is named `statusCompleted`.

Reply:
Oh okay. Then it will need more work.

Reply:
Updated to "statusCompleted".